test/mechanics/timeIntegration/TimeControl.cpp

```diff
@@ -38,55 +38,119 @@ BOOST_AUTO_TEST_CASE(Debug_Fuctionality)
                 };
 
     expectedMsg = "Current timestep is 0 or negative!";
-//    BOOST_CHECK_EXCEPTION(timeControl.SetTimeStepFunction([](double,double)->double{return 0.0;}),
-//                          MechanicsException,CheckException);
-
-
-
+    timeControl.SetTimeStepFunction([](TimeControl&,double,double,bool)->double{return 0.0;});
```

Comment:
The method is defined to take
`std::function<double(TimeControl&, int, int, bool)> timestepFunction`
Since the test is some kind of reference for the usage of the class, this should be correct here (even though it has no influence in this case...).